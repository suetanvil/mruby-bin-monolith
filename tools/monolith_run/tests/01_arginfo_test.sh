#!/bin/bash

set -e

$STAGE/mruby print_arginfo.rb

