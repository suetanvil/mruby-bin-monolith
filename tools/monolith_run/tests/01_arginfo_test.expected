$0: print_arginfo.rb
No MC module!
Done.
